lib/build.js

```diff
@@ -21,26 +21,34 @@ module.exports = {
       enum: [ 'Toggle', 'Keep Visible', 'Show on Error' ],
       order: 1
     },
+    panelOrientation: {
+      title: 'Panel Orientation',
+      description: 'Set where the build panel should be.',
+      type: 'string',
+      default: 'Bottom',
+      enum: [ 'Top', 'Bottom', 'Left', 'Right' ],
```

Comment:
I don't think we should support Left or Right. I can not see any useful scenario where you would like to add it to the left or right of the screen. Since text is left-to-right there would **a lot** of line breaks.

This option should also be added as order 7 to reduce the diff - it is also less likely that people will change this.